docs/content/docs/contributing/Docs.md

```diff
@@ -294,27 +294,29 @@ Markdown content
 
 ### Lists
 
-#### Нумерованный или ненумерованный
+#### Ordered vs Unordered lists
 
-Используй нумерованный список только если он описывает последовательность шагов.
+Only use ordered list for describing an algorithm.
```

Comment:
I need explanation.

Reply:
What about "a sequence of steps" or "step by step guide"? Is it ok?

Reply:
@sboishtyan , ^

Reply:
Explain why I need to use ordered list

Reply:
Replaced by "series of steps to follow"